src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -0,0 +1,1161 @@
+import DialogButton from '../pages/courses/dialog-button'
+import MembershipDialogButton from '../pages/courses/membership-dialog-button'
+
+import LoginForm from 'pages/login'
+
+type CoursePageLayoutProps = {
+  lessons: any
+  course: any
+  ogImageUrl: string
+}
+
+type CollectionResource = {
+  title: string
+  duration: number
+  instructor: {
+    full_name: string
+  }
+  square_cover_url: string
+  image_url: string
+  path: string
+  slug: string
+  description: string
+}
+
+const LessonsList = ({
+  lessons,
+  completedLessonSlugs,
+}: {
+  lessons: any
+  completedLessonSlugs: any
+}) => {
+  return lessons.map((lesson: any, index: number) => {
+    const isComplete = completedLessonSlugs.includes(lesson.slug)
+    return (
+      <li key={lesson.slug}>
+        <div className="flex py-2 font-semibold leading-tight h-20">
+          <div className="flex items-center mr-2 space-x-2">
+            <div
+              className={`${
+                isComplete
+                  ? 'text-blue-600 dark:text-green-400'
+                  : 'text-gray-500 dark:text-gray-400'
+              } pt-px font-xs scale-75 font-normal w-4`}
+            >
+              {isComplete ? (
+                <CheckIcon className="w-6 h-6 -translate-x-2" />
+              ) : (
+                index + 1
+              )}
+            </div>
+            {lesson.icon_url && (
+              <div className="flex items-center flex-shrink-0 w-8">
+                <Image src={lesson.icon_url} width={24} height={24} />
+              </div>
+            )}
+          </div>
+          {lesson.path && (
+            <div className="flex flex-col ">
+              <div>
+                <Link href={lesson.path}>
+                  <a className="text-lg font-semibold hover:underline hover:text-blue-600 dark:text-gray-100">
+                    {lesson.title}
+                  </a>
+                </Link>
+              </div>
+              <div className="text-xs text-gray-700 dark:text-gray-500">
+                {convertTimeWithTitles(lesson.duration, {
+                  showSeconds: true,
+                })}
+              </div>
+            </div>
+          )}
+        </div>
+      </li>
+    )
+  })
+}
+
+const ModulesList = ({modules, render}: {modules: any; render: any}) => {
+  return modules.map((module: any) => {
+    return (
+      <div key={module.id}>
+        <h2 className="text-xl font-bold mt-4 mb-2">{module.title}</h2>
+        {(() => {
+          switch (true) {
+            case module.lessons.length < 5:
+              return (
+                <div className={'grid grid-cols-2'}>
+                  <ul className="grid gap-4">{render(module.lessons)}</ul>
+                  <p className="mb-6 prose text-gray-900 dark:prose-dark md:prose-lg md:dark:prose-lg-dark dark:text-gray-100 dark:prose-a:text-blue-300 dark:hover:prose-a:text-blue-200 prose-a:text-blue-500 hover:prose-a-:text-blue-600 ">
```

Comment:
Description paragraph is currently hardcoded but will later be coming from an external source

Reply:
We are unsure of what the best course of action is to make module descriptions look decent if there is nothing in the empty space. Should the text be wider? I've heard that it's best not to exceed a 75 character width

![image](https://user-images.githubusercontent.com/518406/182922556-b0216e67-0893-42c8-a40f-1c3f6019ce14.png)